AndroidStudioProject/ExeTest.java

```diff
@@ -0,0 +1,42 @@
+package com.example.owner.myapplication;
```

Comment:
This is not an android code - why are we checking in this test file?

Reply:
I'll remove all these files and work in Woongjin's branch.